src/utils/format.ts

```diff
@@ -140,3 +141,21 @@ export function safeFilledToken<T extends TokenDetails>(token: T): T {
     symbol: token.symbol || token.name || abbreviateString(token.address, 6, 4),
   }
 }
+
+export function calculatePriceBigNumber(numerator?: BigNumber, denominator?: BigNumber): BigNumber | null {
+  if (!numerator || !denominator) {
+    return null
+  }
+
+  return numerator.dividedBy(denominator)
```

Comment:
Can `denominator` ever be `0` here? I believe `BN` throws in this instance. Should we check `isZero(denominator)` here?

Reply:
I think they handle that case. Any number divided by 0 is infinity
It's the responsibility of the caller to check that if infinity is not a good number for the use-case.

![image](https://user-images.githubusercontent.com/2352112/72003984-c4444a80-324a-11ea-9937-b790629d9732.png)

Reply:
Also, you can even format that ;)

![image](https://user-images.githubusercontent.com/2352112/72004221-3c127500-324b-11ea-848a-c9d83e4182ac.png)

Reply:
On a second thought, I'm realizing it might not make any sense to return Infinity when showing prices. So it's more convenient to return null, otherwise we have to check `bn.isInfinity()` or ensure the denominator is not 0.

I've made this commit for convenience: cef5704a32e85302e819841248e387058be40002